MNCNH overview and pregnancy component (#1520)
ihmeuw/vivarium_research: docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

> 3.0 Concept model and submodels

Reviewer: Maybe this is just my pet peeve, but isn't this whole _document_ the concept model, so this section should be titled "Concept model diagram" or something?

+++++++++++++++++++++++++++++++

We plan to complete this work in 3 waves. 

> * Wave 1 will include in basic model design, outlines of the healthcare system, and some interventions (AI ultrasound, higher level delivery facility interventions, RDS management). 

Reviewer: ```suggestion
* Wave 1 will include the basic model design, outlines of the healthcare system, and some interventions (AI ultrasound, higher level delivery facility interventions, RDS management). 
```
"in" sounded confusing -- is this what you mean?

> * Wave 2 will some antenatal supplements(MMS, IV iron), the hemoglobin risk for birthing parents, and all downstream causes affected by hemoglobin. 

Reviewer: ```suggestion
* Wave 2 will add in some antenatal supplements(MMS, IV iron), the hemoglobin risk for birthing parents, and all downstream causes affected by hemoglobin. 
```
Missing a verb -- how about this, to match Wave 3?

* Wave 3 will add in gestational blood pressure and relevant causes and risks including pre-eclampsia care and downstream effects of high blood pressure. 

**Wave 1 Concept Model Map:**

> .. image:: wave_1_full.svg

Reviewer: Is there some way to make this diagram narrower and more legible in-line? I have to open it in a new tab and scroll back and forth to read it...

.. _mncnh_portfolio_3.1:

3.1 Model Components
--------------------

Our model will have 3 main "components" that each represent a part of a 

> parent and child dyad's journey. In this model, the simulants (note: simulant 

Reviewer 2: ```suggestion
journey of a parent-child dyad. In this model, the simulants (note: simulant 
```

Reviewer: If you go with Abie's suggestion, I'd also change the last "a" in the previous line to "the," so that it reads "Our model will have 3 main "components" that each represent a part of **the** journey of a parent-child dyad."

> here is a parent child dyad) will each run through 3 decision tree based "time 

Reviewer 2: ```suggestion
here is a parent-child dyad) will each run through 3 decision tree based "time 
```

steps", where future decisions are based on what the simulant previously experienced. 

> For each phase of the model, we will provide inputs needed (state table tracked values), 

Reviewer 2: ```suggestion
For each phase of the model, we will provide inputs needed (in the form of a "state table" of tracked values), 
```

> outputs that need to be recorded, and the decision tree. Nodes in the decision trees are 

Reviewer 2: ```suggestion
outputs that need to be recorded for the next state table, and the decision tree that maps from inputs to outputs. Nodes in the decision trees are 
```

all labeled and additional information will be included below.

3.1.1 Wave 1 Model Components
-----------------------------

**Component 1**: The Pregnancy Model

.. image:: pregnancy_decision_tree_vr.svg

.. list-table:: Pregnancy Decision Tree
  :widths: 3 15 15
  :header-rows: 1

  * - ID
    - Decision Information 
    - Notes
  * - 1
    - XX% of simulants will receive ANC care, based on their propensity 

>     - Several details are still outstanding including: will ANC vary with age, subnational location or upstream factors, will ANC care propensity by correlated with delivery facility propensity

Reviewer: ```suggestion
    - Several details are still outstanding including: will ANC vary with age, subnational location or upstream factors, will ANC care propensity be correlated with delivery facility propensity
```

  * - 2
    - XX% of simulants will receive AI-assisted ultrasound, XX% will receive standard ultrasound, XX% will receive no ultrasound 
    - Need to determine if this is random or correlated with other outcomes. Need to find baseline and scenario values.
  * - 3
    - All patients are assigned a stated gestational age (GA). Their stated GA will be their real GA +/- an error margin randomly assigned within an interval. For AI assisted ultrasound, the interval is +/- XX weeks, for standard ultrasound XX weeks, for no ultrasound XX weeks. 
    - Check if distribution for error is random or more likely to be an over/underestimate. 
  * - 4

>     - XX% of those who received AI-assisted ultrasound will be identified as SGA, XX% with standard ultrasound and XX% with no ultrasound. 

Reviewer: Do we need to include false positive rates here, or will false negative rates be sufficient?

Author: Added to the limitations section

    - 

.. list-table:: Inputs to Pregnancy Decision Tree
  :widths: 3 15 15
  :header-rows: 1

  * - Input
    - Data Source 
    - Notes
  * - Age 
    - GBD and fertility model 
    - Will be the same population generation as used in nutrition optimization pregnancy model 
  * - Upstream factors
    - Likely DHS 
    - Need to decide what if anything we want to include
  * - ANC Visit Propensity
    - Likely DHS 
    - Need to determine correlation if we want to use it 
  * - Gestational age at birth
    - GBD LBWSG
    - 
  * - Small for gestational age at birth 
    - GBD LBWSG

>     - Need to find definition for SGA and determine percent, need to define overlap with gestational age if relevant

Reviewer: For future reference:

https://en.wikipedia.org/wiki/Small_for_gestational_age

https://www.semanticscholar.org/paper/New-Definition-of-Small-for-Gestational-Age-Based-Gardosi/4c2450962d96043a53479719da252b61ad4baa92
